packages/@lwc/engine-server/src/__tests__/fixtures/lifecycle-hooks/expected.html

```diff
@@ -10,6 +10,15 @@
       <li>
         render
       </li>
+      <li>
+        static text
+      </li>
+      <li>
+        static text
+      </li>
+      <li>
+        static text
+      </li>
```

Comment:
Without this change an error is thrown trying to call `toLowerCase` on `childVnode.fragment.tagName` which is undefined.